app-games/src/main/java/com/aptoide/android/aptoidegames/home/GenericErrorView.kt

```diff
@@ -1,92 +1,115 @@
-      Spacer(
-        modifier = Modifier
-          .fillMaxWidth()
-          .weight(177f)
-          .background(
-            brush = blueGradient,
-            alpha = 0.3f
-          )
-      )
-      Spacer(
-        modifier = Modifier
-          .fillMaxWidth()
-          .weight(336f)
-          .background(
-            brush = blueGradient,
-            alpha = 0.08f
-          )
-      )
-      Spacer(
-        modifier = Modifier
-          .fillMaxWidth()
-          .weight(200f)
-          .background(
-            brush = lightGradient,
-            alpha = 0.08f
-          )
-      )
-    }
     Column(horizontalAlignment = Alignment.CenterHorizontally) {
-      Column(modifier = Modifier.weight(288f)) {
-        Spacer(modifier = Modifier.weight(120f))
-        Image(
-          modifier = Modifier.weight(104f),
-          imageVector = AppTheme.icons.ErrorBug,
-          contentDescription = "Error or Bug",
+      Column(modifier = Modifier.weight(312f)) {
+        Spacer(modifier = Modifier.weight(80f))
+        ErrorIconWrapper(
+          modifier = Modifier.weight(144f),
+          errorIcon = imageVector
         )
-        Spacer(modifier = Modifier.weight(64f))
+        Spacer(modifier = Modifier.weight(88f))
       }
-      PaddedRow(sideWeight = 40f) {
-        Text(
+      PaddedRow(sideWeight = 32f) {
+        Column(
           modifier = Modifier.weight(296f),
-          text = stringResource(string.generic_error_message),
-          style = AppTheme.typography.gameTitleTextCondensed,
-          textAlign = TextAlign.Center,
-        )
+          horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+          Text(
+            text = title,
+            style = AppTheme.typography.title,
+            color = AppTheme.colors.onBackground,
+            textAlign = TextAlign.Center,
+          )
+          subtitle?.let {
+            Text(
+              text = it,
+              style = AppTheme.typography.descriptionGames,
+              color = AppTheme.colors.onBackground,
+              textAlign = TextAlign.Center,
+            )
+          }
+        }
       }
-      Spacer(modifier = Modifier.weight(16f))
+      Spacer(modifier = Modifier.weight(24f))
       PaddedRow(sideWeight = 16f) {
         RetryButton(
           onClick = onRetryClick,
-          modifier = Modifier.weight(344f)
+          modifier = Modifier.weight(328f)
         )
       }
-      Spacer(modifier = Modifier.weight(296f))
+      Spacer(modifier = Modifier.weight(154f))
+      Image(
+        imageVector = getChessPatternBanner(),
+        contentDescription = null,
+        modifier = Modifier.fillMaxWidth(),
+        contentScale = ContentScale.FillWidth
+      )
     }
   }
 }
 
+@Composable
+fun GenericErrorView(
+  onRetryClick: () -> Unit,
+  modifier: Modifier = Modifier,
+) {
+  ErrorView(
+    imageVector = AppTheme.icons.Bug,
+    title = stringResource(R.string.generic_error_message),
+    subtitle = null,
```

Comment:
This should be separated, right?
The title should only be "Something went wrong." and the subtitle should be "Try again."